modin/engines/base/io/text/text_file_dispatcher.py

```diff
@@ -258,3 +259,157 @@ def _read_rows(
             rows_read += 1
 
         return outside_quotes, rows_read
+
+    # _read helper functions
+    @classmethod
+    def rows_skipper_builder(cls, f, quotechar, is_quoting):
+        def skipper(n):
+            if n == 0 or n is None:
+                return 0
+            else:
+                return cls._read_rows(
+                    f,
+                    quotechar=quotechar,
+                    is_quoting=is_quoting,
+                    nrows=n,
+                )[1]
+
+        return skipper
+
+    @classmethod
+    def _define_header_size(
+        cls,
+        header: Union[int, Sequence[int], str, None] = "infer",
+        names: Union[Sequence, None] = None,
+    ):
+        """
+        Defines the number of rows that are used by header.
+        Parameters
+        ----------
+        header: int, list of int or str ("infer")
+                Original header parameter of read_csv function.
+        names:  array
+                Original names parameter of read_csv function.
+        Returns
+        -------
+        header_size:
+                The number of rows that are used by header.
+        """
+        header_size = 0
+        if header == "infer" and names is None:
+            header_size += 1
+        elif isinstance(header, int):
+            header_size += header + 1
+        elif hasattr(header, "__iter__") and not isinstance(header, str):
+            header_size += max(header) + 1
+
+        return header_size
+
+    @classmethod
+    def _define_usecols(
+        cls,
+        filepath_or_buffer: FilePathOrBuffer,
+        **kwargs,
+    ):
```

Comment:
Could we add a type hint for the return type here as well?

Reply:
Ok,  return type hint was added.